deluge/ui/console/commands/config.py

```diff
@@ -65,14 +65,14 @@ def simple_eval(source):
 class Command(BaseCommand):
     """Show and set configuration values"""
 
-    usage = "config [--set <key> <value>] [<key> [<key>...] ]"
+    usage = _("Usage: config [--set <key> <value>] [<key> [<key>...] ]")
```

Comment:
Yes I did ponder that and it's a good point about the `key` and `value`... I'll have a think :)